src/includes/sentry-cli-sourcemaps.mdx

```diff
@@ -10,16 +10,12 @@ For more info on `sentry-cli` configuration visit the [Sentry CLI configuration
 
 Make sure `sentry-cli` is configured for your project. For that you can use environment variables:
 
-<Note>
-
-Visit the [auth token user settings page](https://sentry.io/settings/account/api/auth-tokens/) to find your auth tokens.
-
-</Note>
+<OrgAuthTokenNote />
 
 <SignInNote />
```

Comment:
```suggestion
```

OrgAuthTokenNote is kind of a more detailed version of SignInNote, so when we show this we can skip the sign in note!

Reply:
The sign in note needs to come before every instance where we use `___ORG_SLUG___` or `___PROJECT_SLUG___`.

Reply:
With https://github.com/getsentry/sentry-docs/pull/7716 we can make this all a bit more cleaner